ocs_ci/ocs/ui/pvc_ui.py

```diff
@@ -51,9 +52,52 @@ def create_pvc_ui(self, sc_type, pvc_name, access_mode, pvc_size):
         logger.info("Select PVC size")
         self.do_send_keys(self.pvc_loc["pvc_size"], text=pvc_size)
 
+        if (
+            sc_type
+            in ("ocs-storagecluster-ceph-rbd-thick", "ocs-storagecluster-ceph-rbd")
+            and access_mode == "ReadWriteOnce"
+        ):
+            logger.info(
+                "Test running on OCP version" + ":" + str({get_running_ocp_version()})
```

Comment:
```suggestion
                f"Test running on OCP version: {get_running_ocp_version()}"
```
Didn't get your origin formatting.

Reply:
updated with f string as suggested